.github/pull_request_template.md

```diff
@@ -0,0 +1,29 @@
+
+<!--
+# Thanks for contributing a pull request! 
+## Please make sure you see our contribution guidelines: https://github.com/narwhals-dev/narwhals/blob/main/CONTRIBUTING.md
+-->
+
+## What type of PR is this? (check all applicable)
+
+- [ ] 💾 Refactor
+- [ ] ✨ Feature
+- [ ] 🐛 Bug Fix
+- [ ] 🔧 Optimization
+- [ ] 📝 Documentation
+- [ ] ✅ Test
+- [ ] 🐳 Other
+
+## Related Issues 
```

Comment:
```suggestion
## Related issues 
```
Let's keep the formatting of the heading consistent with the rest of the template.

Reply:
Thanks. I'm not sure why I kept doing that.